cli/tests/end_to_end/extension.rs

```diff
@@ -75,6 +75,33 @@ pub async fn get_groups() {
         .stdout(predicates::str::contains("groups"));
 }
 
+#[tokio::test]
+pub async fn create_and_delete_project() {
+    let test_cli = TestCli::builder().with_config(None).build();
+
+    test_cli
+        .extension(
+            r#"
+            try {
+                await PhylumApi.deleteProject("create_and_delete_test_project")
+            } catch (e) {
+            }
+
+            let projectId = await PhylumApi.createProject("create_and_delete")
+            let duplicatedProjectId = await PhylumApi.createProject("create_and_delete")
```

Comment:
I don't understand, why are we deleting `create_and_delete_test_project` here, when `create_and_delete` is the project name we use for the test? I assumed this was just a setup to ensure a clean state, but this wouldn't actually delete the project.

Reply:
It was indeed a typo.